packages/lit-dev-content/site/tutorials/content/first-component/06.md

```diff
@@ -0,0 +1,140 @@
+In this section we'll make it so when you click on the `word-viewer` the
+animation plays in reverse.
+
+Before adding the event handling we need a new piece of internal state to
+represent the current animation `playDirection`.
+
+{% switchable-sample %}
+
+```ts
+// word-viewer.ts
+
+  @state() private playDirection = 1;
+
+```
+
+```js
+// word-viewer.js
+
+  static properties = {
+    playDirection: {state: true},
+    ...
+  };
+
+  constructor() {
+    ...
+    this.playDirection = 1;
+  }
+
+```
+
+{% endswitchable-sample %}
+
+Next the math in `render()` to find the `word` needs to be updated to account
+for a negative `this.idx`.
+
+{% switchable-sample %}
+
+```ts
+// word-viewer.ts
+
+  render() {
+    ...
+    const word = splitWords[((this.idx % splitWords.length) + splitWords.length) % splitWords.length];
+    ...
+  }
+
+```
+
+```js
+// word-viewer.js
+
+  render() {
+    ...
+    const word = splitWords[((this.idx % splitWords.length) + splitWords.length) % splitWords.length];
+    ...
+  }
+
+```
+
+{% endswitchable-sample %}
+
+Change `tickToNextWord` to choose the next word based on the `playDirection`.
+
+{% switchable-sample %}
+
+```ts
+// word-viewer.ts
+
+  tickToNextWord = () => { this.idx += this.playDirection; };
+
+```
+
+```js
+// word-viewer.js
+
+  tickToNextWord = () => { this.idx += this.playDirection; };
+
+```
+
+{% endswitchable-sample %}
+
+And add a `switchPlayDirection` method to invert the  `playDirection`.
+
+
+{% switchable-sample %}
+
+```ts
+// word-viewer.ts
+
+  switchPlayDirection() {
+    this.playDirection *= -1;
+  }
+
+```
+
+```js
+// word-viewer.js
+
+  switchPlayDirection() {
+    this.playDirection *= -1;
+  }
+
+```
+
+{% endswitchable-sample %}
+
+Finally, invoke `this.switchPlayDirection` when the `<pre>` element is clicked
+by using Lit's [declarative event listener
+syntax](/docs/components/events/#adding-event-listeners-in-the-element-template).
+
+{% switchable-sample %}
+
+```ts
+// word-viewer.ts
+
+  render() {
+    ...
+    return html`<pre
+      @click=${this.switchPlayDirection}
+    >${word}</pre>`;
+  }
+
+```
+
+```js
+// word-viewer.js
+
+  render() {
+    ...
+    return html`<pre
+      @click=${this.switchPlayDirection}
+    >${word}</pre>`;
+  }
+
+```
+
+{% endswitchable-sample %}
+
+When clicking the `word-viewer` in the preview, the words should reverse their
```

Comment:
Infinitive case of "When clicking" doesn't make sense here. It reverses after, not during the click. Maybe make it an action

> Click the `word-viewer` in the preview, the word animation should reverse in direction.

or something like that